packages/@webex/contact-center/src/services/task/TaskManager.ts

```
@@ -538,6 +569,47 @@ export default class TaskManager extends EventEmitter {
     }
   }
 
+  /**
+   * Handles auto-answer logic for incoming tasks
+   * Automatically accepts tasks when isAutoAnswering flag is set
+   * The flag is set during task creation based on:
+   * 1. WebRTC calls with auto-answer enabled in agent profile
+   * 2. Agent-initiated WebRTC outdial calls
+   * 3. Agent-initiated digital outbound (Email/SMS) without previous transfers
+   *
+   * @param task - The task to auto-answer
+   * @private
+   */
+  private async handleAutoAnswer(task: ITask): Promise<void> {
+    if (!task || !task.data || !task.data.isAutoAnswering) {
+      return;
```

Maybe instead of returning, we throw an error here?

---

Kept as return — this guard clause handles the normal operation where most tasks don’t have auto-answer enabled. Throwing an error here would treat the expected path as exceptional and introduce unnecessary error handling in the callers.